README.md

```diff
@@ -59,6 +59,19 @@ See the INSTALL file for details; in brief, it uses the `configure`,
 check out the INSTALL file for some guidance on flags, or the output
 of `./configure --help`.
 
+An example configuration line that picks up packages installed at
+various paths might look like the following.  Note that depending on
+the package, the --with-<pkg>= directive expects slightly different
+things.  hdf5 wants a compiler wrapper; blas and lapack want lists of
+linker flags; elemental and gtest want an installation prefix.
+
+./configure --with-CXX=<mpipath>/bin/mpic++ \
+            --with-hdf5=<hdf5path>/bin/h5cc \
+	    --with-blas='-L<blaspath> -lopenblas' \
+	    --with-lapack='-L<blaspatch> -lopenblas' \
+	    --with-elemental=<elpath> \
```

Comment:
Given PR #12, it's probably time to strip out the [Elemental](https://github.com/elemental/Elemental) support -- Elemental is deprecated and the LLNL fork strips out all functionality except for BLAS Level 1 subroutines. We don't need to make this change in this PR -- I'll add a separate issue.